src/main/java/org/skriptlang/skript/bukkit/loottables/elements/effects/EffGenerateLoot.java

```diff
@@ -0,0 +1,93 @@
+package org.skriptlang.skript.bukkit.loottables.elements.effects;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.SyntaxStringBuilder;
+import ch.njol.util.Kleenean;
+import org.bukkit.Bukkit;
+import org.bukkit.event.Event;
+import org.bukkit.inventory.Inventory;
+import org.bukkit.loot.LootContext;
+import org.bukkit.loot.LootTable;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.bukkit.loottables.LootContextWrapper;
+
+import java.util.Random;
+import java.util.concurrent.ThreadLocalRandom;
+
+@Name("Generate Loot")
+@Description({
+	"Generates the loot in the specified inventories from a loot table using a loot context. " +
+	"Not specifying a loot context will use a loot context with a location at the world's origin.",
+	"Note that if the inventory is full, it will cause warnings in the console due to over-filling the inventory."
```

Comment:
```suggestion
	"Generates the loot in the specified inventories from a loot table using a loot context. "
		+ "Not specifying a loot context will use a loot context with a location at the world's origin.",
	"Note that if the inventory is full, it will cause warnings in the console due to over-filling the inventory."
```